doc/_extensions/lexer/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) 2021 Zephyr Project members and individual contributors
```

Comment:
Not Nordic?

Reply:
These are just comments to pass compliance checks. As the files were already there I thought it was better to add this copyright message.

Reply:
i guess nobody wants to own copyright on an empty file :-D